tests/Support/InitializesSettings.php

```diff
@@ -10,8 +10,15 @@ trait InitializesSettings
 
     public function initializeSettings()
     {
+        if (app()->environment('testing')) {
+            if (class_exists(\Database\Seeders\DemoSeeder::class)) {
+                putenv('SEED_DEMO=false');
+                config(['snipeit.seed_demo' => false]);
+            }
+        }
+
```

Comment:
This should be able to be removed. We don't seed in tests and have each one build up the world via factories.